schemas/view-schema.json

```diff
@@ -219,6 +219,36 @@
                 "readOnly": true
               },
               "readOnly": true
+            },
+            "memory_average": {
+              "type": "number",
+              "description": "Average memory used during the view lifetime (in bytes)",
+              "readOnly": true
+            },
+            "memory_max": {
+              "type": "number",
+              "description": "Peak memory used during the view lifetime (in bytes)",
+              "readOnly": true
+            },
+            "cpu_ticks_count": {
+              "type": "number",
+              "description": "Total number of cpu ticks during the view’s lifetime",
+              "readOnly": true
+            },
+            "cpu_ticks_per_second": {
+              "type": "number",
+              "description": "Average number of cpu ticks per second during the view’s lifetime",
+              "readOnly": true
+            },
+            "refresh_rate_average": {
```

Comment:
not sure if this one will be useful, especially if view is opened for a long time, but will see.

from practical point of view the most of the freezes/lags happen in the first 3-5 seconds since view is opened, probably - when everything is created from the scratch and data is loaded.

same probably applies to CPU, but with less degree - because of the background work.

Reply:
Yes but on average the duration on a given View on mobile can be short. Most of the time if the view spans over a long time it will probably have a few lags (e.g. infinite scroll on posts in a social media app where you're staying on the same activity and load new content every once in a while)